experimental/dds/tree2/src/test/shared-tree-core/editManagerTestUtils.ts

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	ChangeFamily,
+	SessionId,
+	ChangeRebaser,
+	ChangeFamilyEditor,
+	mintRevisionTag,
+} from "../../core";
+import {
+	TestAnchorSet,
+	TestChangeFamily,
+	TestChange,
+	testChangeFamilyFactory,
+	TestChangeRebaser,
+} from "../testChange";
+import { MockRepairDataStoreProvider } from "../utils";
+import { EditManager } from "../../shared-tree-core";
+import { brand } from "../../util";
+
+export type TestEditManager = EditManager<ChangeFamilyEditor, TestChange, TestChangeFamily>;
+
+export function editManagerFactory(options: {
+	rebaser?: ChangeRebaser<TestChange>;
+	sessionId?: SessionId;
+}): {
+	manager: TestEditManager;
+	anchors: TestAnchorSet;
+	family: ChangeFamily<ChangeFamilyEditor, TestChange>;
+} {
+	const family = testChangeFamilyFactory(options.rebaser);
+	const anchors = new TestAnchorSet();
+	const manager = new EditManager<
+		ChangeFamilyEditor,
+		TestChange,
+		ChangeFamily<ChangeFamilyEditor, TestChange>
+	>(family, options.sessionId ?? "0", new MockRepairDataStoreProvider(), anchors);
+	return { manager, anchors, family };
+}
+
+export function rebaseLocalEditsOverTrunkEdits(
+	nbLocal: number,
```

Comment:
There are a lot of `nb`s in this PR - I'm assuming that stands for "number"? I think I prefer "Count" (suffix) for readability, but if `nb` is a convention that has bled up from rebaser code or something, I'm fine with it. It's nice to have consistency, though (should we use `nb` elsewhere in shared-tree-core for counts of things?).

Reply:
I don't think `nb` has been used that much in the rebaser code. I'll change this to use `count`.